packages/react/src/autocomplete/Autocomplete.test.tsx

```diff
@@ -55,16 +55,17 @@ describe('Autocomplete', () => {
 		if (!input) return;
 
 		// Focus the input and start typing a search term
-		input.focus();
-		userEvent.type(input, 'qld');
+		act(() => input.focus());
```

Comment:
This should fix that pesky console.warning we sometimes get when running tests locally.